### ISSUE TITLE
`sklearn` addition of `normalized_stress` parameter to `manifold.MDS`

### ISSUE THREAD

**Issue by RobertSamoilescu** [SeldonIO/alibi#854]
`sklearn 1.2` included an new parameter, `normalized_stress`, to the [`manifold.MDS`](https://scikit-learn.org/stable/modules/generated/sklearn.manifold.MDS.html#sklearn.manifold.MDS). Until 1.4., the default value is `warn` which will print a warning, but then replaced by `False`.  Following 1.4. the default value will be `False` which is actually the intended behavior. Thus, most likely no code base change will be required.
